File: js/ejer56.js
//simular carrito de compras (agregar productos y calcular total)
const carrito = [];

function agregarProducto(nombre, precio) {
    carrito.push({ nombre, precio });
}   

function calcularTotal() {
    return carrito.reduce((total, producto) => total + producto.precio, 0);
}
agregarProducto('Camisa', 25);
agregarProducto('Pantalón', 40);
agregarProducto('Zapatos', 60); 
console.log('Total a pagar: $' + calcularTotal());
        

File: python/ejer56.py
#simular carrito de compras (agregar productos y calcular total)
class Carrito:
    def __init__(self):
        self.productos = []
    
    def agregar_producto(self, nombre, precio):
        self.productos.append({'nombre': nombre, 'precio': precio})
    
    def calcular_total(self):
        total = sum(producto['precio'] for producto in self.productos)
        return total
# Ejemplo de uso
carrito = Carrito() 
carrito.agregar_producto('Manzana', 1000)
carrito.agregar_producto('Pan', 400)
total = carrito.calcular_total()
print(f'Total a pagar: ${total:.2f}')
